File: Atividade_pratica/modulo2/cadastro_agencia.py
def cadastro_cliente (): 
        
    nome = input("Digite seu nome:");
    idade = input("Digite sua idade:");
    cpf = input("Digite seu cpf:");
    endereco = input("Digite seu endereço:");
    pessoa = [];
    pessoa.append(nome);
    pessoa.append(idade);
    pessoa.append(cpf);
    pessoa.append(endereco);
    consulta = input("Deseja consultar cliente: ")

def cadastro_destino():
    destino = input("Digite a cidade de destino:");
    periodo = input("Digite os dias que passará no destino: ");
    data = input("Digite a data de saída para o destino:");

cadastro = int(input("Digite 1 para cadastrar cliente, 2 para cadastrar destino:"));

if (cadastro == 1):
    cadastro_cliente();
if (cadastro == 2):
    cadastro_destino();
consulta = int(input("Digite 3 para consultar cliente, 4 para consultar destino:"));

if (consulta == 3):
    print(pessoa);
if (consulta == 4):
    print(cadastro_destino);

File: Atividade_pratica/modulo2/codigo_agencia/cadastro_agencia.js

    const clientes = [];
    
    function cadastrarCliente() {

        const cliente = {};
    
        const nome = document.getElementById("Nome").value;
        const idade = document.getElementById("Idade").value;
        const cpf = document.getElementById("CPF").value;
        const endereco = document.getElementById("Endereco").value;

        cliente.Nome = nome;
        cliente.Idade = idade;
        cliente.CPF = cpf;
        cliente.Endereço = endereco;

        clientes.push(cliente); 
        
        document.getElementById("Nome").value = "";
        document.getElementById("Idade").value = "";
        document.getElementById("CPF").value = "";
        document.getElementById("Endereco").value = "";

    }

   

    function consultarCliente(){
        const resultadoElement = document.getElementById("resultado");
        resultadoElement.innerText = JSON.stringify(clientes, null, 2);
    }
  
    

    const destinos = [];
    
    function cadastrarDestino() {

        const destino = {};
    
        const cidade = document.getElementById("Cidade").value;
        const periodo = document.getElementById("Periodo").value;
        const data = document.getElementById("Data").value;
       

        destino.Cidade = cidade;
        destino.Período = periodo;
        destino.Data = data;
        

        destinos.push(destino); 
        
        document.getElementById("Cidade").value = "";
        document.getElementById("Periodo").value = "";
        document.getElementById("Data").value = "";
        

    }

   

    function consultarDestino(){
        const resultadoElement = document.getElementById("resultado");
        resultadoElement.innerText = JSON.stringify(destinos, null, 2);
    }
  
    
